types/crpc/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
```

Comment:
The CI is complaining because you didn't enable `strictFunctionTypes`.

Reply:
Handled this, will open new PR with change.